example/javalib/java-home/1-downloading-jvms/build.mill

```diff
@@ -0,0 +1,29 @@
+// :link-jvm-indices: https://github.com/coursier/jvm-index/tree/master/indices
```

Comment:
Let's also include a version of the `downloading-jvms` example for scalalib and kotlinlib